pyproject.toml

```diff
@@ -1,19 +1,50 @@
+[tool.bandit]
+exclude_dirs = ["/venv/"]
+[tool.bandit.assert_used]
+skips = ["*/*test.py", "*/test_*.py", "*tests/*.py"]
+
 # Testing tools configuration
 [tool.coverage.run]
 branch = true
 
+# Formatting tools configuration
+[tool.black]
+line-length = 99
+target-version = ["py38"]
+
 [tool.coverage.report]
 show_missing = true
 
+# Linting tools configuration
+[tool.flake8]
+max-line-length = 99
+max-doc-length = 99
+max-complexity = 10
+exclude = [".git", "__pycache__", ".tox", "build", "dist", "*.egg_info", "venv"]
+select = ["E", "W", "F", "C", "N", "R", "D", "H"]
+# Ignore W503, E501 because using black creates errors with this
+# Ignore D107 Missing docstring in __init__
+ignore = ["W503", "E501", "D107"]
+# D100, D101, D102, D103: Ignore missing docstrings in tests
+per-file-ignores = ["tests/*:D100,D101,D102,D103,D104,D205,D212,D415"]
+docstring-convention = "google"
+
+[tool.isort]
+line_length = 99
+profile = "black"
+
+[tool.mypy]
+ignore_missing_imports = true
+explicit_package_bases = true
+namespace_packages = true
+
+[tool.pylint]
+disable = "wrong-import-order"
+
 [tool.pytest.ini_options]
 minversion = "6.0"
 log_cli_level = "INFO"
 
-# Formatting tools configuration
-[tool.black]
-line-length = 99
-target-version = ["py38"]
-
```

Comment:
May I know what is the reason for removing this line? From what I understand, this is to have compatibility with VSCode black formatter setup in contributing guide :D

Reply:
Its still in the file, just moved up.

Reply:
It has been not removed. Now it's on line 10 ;)